src/components/configurable-items/configurable-items.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+
+import { ConfigurableItems, ConfigurableItemRow } from ".";
+import Button from "../button";
+import Box from "../box";
+
+<Meta title="Configurable Items" parameters={{ info: { disable: true } }} />
+
+# Configurable Items
+
+- This component renders a list of items that can be checked/unchecked and re-ordered.
+- Designed to be used in tandem with the onConfigure prop in the table component so that columns can be enabled/disabled and re-ordered.
+
+## Quick Start
+
+```jsx
+import {
+  ConfigurableItems,
+  ConfigurableItemRow,
+} from "carbon-react/lib/components/configurable-items";
+```
+
+### Default
+
+**To see proper functioning of this component please open it in `Canvas` (click on `Canvas` in top tab).**
+
+<Preview>
+  <Story name="default">
+    {() => {
+      const defaultConfigurableItemsData = [
+        {
+          enabled: true,
+          locked: true,
+          name: "test 1",
+          key: "1",
+        },
+        {
+          enabled: true,
+          locked: false,
+          name: "test 2",
+          key: "2",
+        },
+        {
+          enabled: false,
+          locked: false,
+          name: "test 3",
+          key: "3",
+        },
+      ];
+      const [data, setData] = useState(defaultConfigurableItemsData);
+      const handleDrag = (draggedIndex, hoveredIndex) => {
+        setData((prevData) =>
+          prevData.map((item, index) => {
+            if (index === draggedIndex) {
+              return data[hoveredIndex];
+            }
+            if (index === hoveredIndex) {
+              return data[draggedIndex];
+            }
+            return item;
+          })
+        );
+      };
+      const handleChange = (rowIndex) => {
+        setData((prevData) =>
+          prevData.map((item, index) =>
+            index === rowIndex ? { ...item, enabled: !item.enabled } : item
+          )
+        );
+      };
+      const handleSave = (event) => {
+        event.persist();
+      };
+      const handleReset = () => {
+        setData(defaultConfigurableItemsData);
+      };
+      return (
+        <ConfigurableItems
+          onDrag={handleDrag}
+          onCancel={() => console.log("cancel")}
```

Comment:
Does this `console.log` pass linting without a disable line comment?

Reply:
It's in MDX file, so ESlint does not work here :)